### ISSUE TITLE
Drop the systemd module dependency?

### ISSUE THREAD

**Issue by walkamongus** [jsok/puppet-vault#137]
I know some convenience might be lost, but specifying a systemd module as a dependency seems to over-reach as far as module scope is concerned. The dependency forces this module's users into using a specific module to manage systemd, which has system-wide effects.

**Comment by Pixelshifter** [comment]
Agreed. I was just looking into upgrading this module to 1.4 for Puppet 6 but this seems not very appealing.
